test/integration/api/forms/forms.js

```diff
@@ -724,7 +725,7 @@ describe('api: /projects/:id/forms (create, read, update)', () => {
             .set('Content-Type', 'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet')
             .expect(200)
             .then(() => service.login('chelsea', (asChelsea) =>
-              asChelsea.get('/v1/projects/1/forms/simple2.xlsx')
+              asChelsea.get('/v1/projects/1/forms/simple2/draft.xlsx')
```

Comment:
I don't think these lines needed to change... but it's also confusing because it's testing good ol' Chelsea's access and she doesn't have access to anything, so of course there's going to be some kind of 400 error regardless of if we're looking at the draft or not.

Reply:
:+1: reverted